plugins/mmpstrucdata/mmpstrucdata.c

```diff
@@ -216,7 +216,7 @@ parseSD_NAME(uchar *sdbuf, int lenbuf, int *curridx, uchar *namebuf)
 		if(   sdbuf[i] == '=' || sdbuf[i] == '"'
 		   || sdbuf[i] == ']' || sdbuf[i] == ' ')
 			break;
-		namebuf[j] = tolower(sdbuf[i]);
+		namebuf[j] = sdbuf[i];
```

Comment:
We cannot do this unconditionally as others may depend on existing behavior. So we need to introduce an option to keep the lowercasing or not.

Reply:
I think that's beyond my coding abilities - if you can give me some pointers I may be able to add the option.

Reply:
If you don't mind, I would add it -- it's just a couple of minutes for me as I know exactly where to place it. Actually quicker than to describe it with words. I could post the resulting PR for your review, so you know the next time (if any). What do you think?

Reply:
I take the non-response as a go-ahead ;-)